eth-supply/proto/v1/eth_supply.proto

```diff
@@ -0,0 +1,11 @@
+syntax = "proto3";
+
+package messari.eth_supply.v1;
+
+message EthSupply {
+  string genesis = 1;      // string: BigInt, in wei
```

Comment:
Is there a reason why you're not using a BigInt type here and are using string instead?

Reply:
Yeh you can just define your own as a bytes object (It's been done like that for some of the other substreams for if you want to see an example).